### ISSUE TITLE
Minecraft 1.21.x support?

### ISSUE THREAD

**Issue by peti** [EngineHub/WorldEditCUI#119]
### The Problem

It would be great to be able to use this mod with the latest version of Minecraft 1.21.x. I see that there is a port for NeoForge out there, but I'd rather stay with Fabric if possible (and mixing the two breaks my shaders). 

### A Solution

Port the code to Minecraft 1.21.x, I guess? :smile:

**Comment by me4502** [comment]
duplicate of https://github.com/EngineHub/WorldEditCUI/issues/116

the release is held up by a fabric issue
